src/java/org/apache/cassandra/schema/CompressionParams.java

```diff
@@ -605,6 +605,7 @@ static class Serializer implements IVersionedSerializer<CompressionParams>
     {
         public void serialize(CompressionParams parameters, DataOutputPlus out, int version) throws IOException
         {
+            assert version >= MessagingService.VERSION_40;
```

Comment:
It's not related to the changes but, since we are here, we can remove the unneeded class-level `@SuppressWarnings("deprecation")` annotation. That existed due to the deprecated properties `SSTABLE_COMPRESSION`, `CHUNK_LENGTH_KB` and `CRC_CHECK_CHANCE`, which aren't here anymore. Feel free to ignore.